src/js/components/FilterCriteria.jsx

```diff
@@ -10,6 +10,7 @@ import {operators} from '../constants'
 import RaisedButton from 'material-ui/RaisedButton'
 import {SelectField} from 'safe-framework'
 import TextField from 'material-ui/TextField'
+import TooltipComponent from './TooltipComponent'
```

Comment:
I would prefer this to just be called `Tooltip`. So far, only a few components actually have `Component` in the name, and they should probably be changed anyway.